tests/test_transmitter.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2014-2023 XMOS LIMITED.
+# This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+import pytest
+import Pyxsim
+from Pyxsim  import testers
+from pathlib import Path
+from spdif_test_utils import (
+    Spdif_rx,
+    Frames,
+)
+
+MAX_CYCLES = 15000000
+
+def _get_duration():
+    return 193
+
+def _get_mclk_freq(sam_freq):
+    if sam_freq in [48000,96000,192000]:
+        return 24576000
+    elif sam_freq in [44100,88200,176400]:
+        return 22579200
```

Comment:
it would be good to test with different mclk freqs - we should just add an issue for now.